app/models/profile.rb

```diff
@@ -33,6 +33,7 @@ def decrypt_pii(user_access_key)
   end
 
   def recover_pii(personal_key)
+    personal_key = personal_key.tr('-', ' ')
```

Comment:
@monfresh I like that idea. Could move the `normalize` method into the new `PersonalKeyValidator` since it's only relevant for incoming strings.